src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
Let's do so some clearing 🙂  try to remove all unnecessary code or file;
![image](https://user-images.githubusercontent.com/31897434/92557006-24204480-f263-11ea-83b8-bb7a5a7ff6db.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/

Reply:
Done.
Thanks for your suggestion bellom! :).